### ISSUE TITLE
Asynchronous Methods for Deep Reinforcement Learning

### ISSUE THREAD

**Issue by joyhuang9473** [number9473/nn-algorithm#246]
# Asynchronous Methods for Deep Reinforcement Learning #

- Author: Volodymyr Mnih, Adrià Puigdomènech Badia, Mehdi Mirza, Alex Graves, Timothy P. Lillicrap, Tim Harley, David Silver, Koray Kavukcuoglu
- Origin: https://arxiv.org/abs/1602.01783
- Related:
  - https://github.com/MorvanZhou/Reinforcement-learning-with-tensorflow/blob/master/contents/10_A3C/A3C_continuous_action.py
  - https://github.com/MorvanZhou/pytorch-A3C

**Comment by joyhuang9473** [comment]
莫烦: Asynchronous Advantage Actor-Critic (A3C) (Tensorflow)
https://morvanzhou.github.io/tutorials/machine-learning/reinforcement-learning/6-3-A3C/

**Comment by joyhuang9473** [comment]
#247
